tests/Auth0Net.DependencyInjection.Tests/System.Runtime.CompilerServices/IsExternalInit.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Original source code copied from
```

Comment:
I'd prefer the use of Simon Cropp's [Polyfill ](https://github.com/SimonCropp/Polyfill )package than having this in the codebase.